cassandra-win-config.yml.sample

```
@@ -0,0 +1,40 @@
+integration_name: com.newrelic.cassandra
+
+instances:
+    - name: cassandra-metrics
+      command: metrics
+      arguments:
+          hostname: localhost
+          port: 7199
+          username: testUser
+          password: testPassword
+
+          # New users should leave this property as `true`, to identify the
+          # monitored entities as `remote`. Setting this property to `false` (the
+          # default value) is deprecated and will be removed soon, disallowing
+          # entities that are identified as `local`.
+          # Please check the documentation to get more information about local
+          # versus remote entities:
+          # https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/entity-definition.md
+          remote_monitoring: true
+          NR_JMX_TOOL: "C:\\PROGRA~1\\NEWREL~1\\nrjmx\\nrjmx.bat"
+      labels:
+          env: production
+          role: cassandra
+    - name: cassandra-inventory
+      command: inventory
+      arguments:
+          config_path: c:\apache-cassandra-3.11.9\conf\cassandra.yaml
```

`hostname` needs to be added for the entity name and inventory workaround.
```suggestion
          hostname: localhost
          config_path: c:\apache-cassandra-3.11.9\conf\cassandra.yaml
```

---

did not know that, thanks